modules/criteoBidAdapter.js

```diff
@@ -156,18 +156,24 @@ export const spec = {
    * @param {TimedOutBid} timeoutData
    */
   onTimeout: (timeoutData) => {
-    if (publisherTagAvailable()) {
-      // eslint-disable-next-line no-undef
-      const adapter = Criteo.PubTag.Adapters.Prebid.GetAdapter(timeoutData.auctionId);
-      adapter.handleBidTimeout();
+    if (publisherTagAvailable() && timeoutData.length > 0) {
```

Comment:
I am sure prebid core does the right thing and passes an array ALWAYS here, but just to be safe, a check for Array.isArray here may be a good idea?

In case someone accidentally does something and messes up prebid core?

Reply:
Oki, done